ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/response/FullAdmissionResponseMapperTest.java

```diff
@@ -76,8 +73,10 @@ public void shouldMapFullAdmissionResponseFromCCD() {
         //given
         CCDFullAdmissionResponse ccdFullAdmissionResponse = CCDFullAdmissionResponse.builder()
             .moreTimeNeededOption(CCDYesNoOption.YES)
-            .paymentOption(BY_SPECIFIED_DATE)
-            .paymentDate(LocalDate.now().plusDays(7))
+            .paymentIntention(CCDPaymentIntention.builder()
```

Comment:
can use `SamplePaymentIntention.bySetDate()`

Reply:
We do not have sample builder for `CCDPaymentIntention` :(

Reply:
correct